eva-server/src/main/java/uk/ac/ebi/eva/server/Profiles.java

```diff
@@ -0,0 +1,9 @@
+package uk.ac.ebi.eva.server;
+
+/**
+ * Created by jorizci on 01/11/16.
+ */
+public class Profiles {
+    public static final String OAUTH_SECURITY = "oauth2-security";
+    public static final String NOT_OAUTHSECURITY = "!"+ OAUTH_SECURITY;
```

Comment:
Ok, a little explanation, NOT_OAUTHSECURITY is not a profile, it means oauth-security profile is not enabled, so the condition if we add more system will be NOT_SEC_A and NOT_SEC_B, etc... That makes that by default if you don't have to modify the configuration to use the unsecure configuration.